phpseclib/Net/SSH2_CHANNEL_OPEN_FAILURE_REASONS.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace phpseclib\Net;
+
+use phpseclib\Common\Enum;
+
+class SSH2_TERMINAL_MODES extends Enum
+{
+    const TTY_OP_END=0;
+}
```

Comment:
The class name (and constants defined therein) should probably be SSH2_CHANNEL_OPEN_FAILURE_REASONS lol